src/index.html

```diff
@@ -17,6 +17,12 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+
+    <div class="stopwatch stopwatch--speed-up">
+      <div class="stopwatch__minutes-hand"></div>
+      <div class="stopwatch__seconds-hand"></div>
+      <div class="stopwatch__center"></div>
```

Comment:
BEM naming convention suggests that modifiers should be used with the name of the block or element. 'stopwatch--speed-up' seems to be a modifier, but it's not clear which block or element it's modifying. Ensure that it follows the BEM naming convention, for example, 'stopwatch stopwatch--speed-up' if 'speed-up' is a state of 'stopwatch'.